<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SFV Moveset Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>BAC</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="BAC_Name">
       <property name="text">
        <string>BAC File</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QVBoxLayout" name="BAC_Buttons_Layer">
       <item>
        <widget class="QPushButton" name="EditBAC">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Edit BAC</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>BCM</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="BCM_Name">
       <property name="text">
        <string>BCM File</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <layout class="QVBoxLayout" name="BCM_Buttons_Layer">
       <item>
        <widget class="QPushButton" name="EditBCM">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Edit BCM</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>BAC EFF</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="BAC_Eff_Name">
       <property name="text">
        <string>BAC_Eff File</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QVBoxLayout" name="BAC_Eff_Buttons_Layer">
       <item>
        <widget class="QPushButton" name="EditBACEff">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Edit BAC EFF</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Date_Time">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>41</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDate|Qt::ImhTime</set>
    </property>
    <property name="text">
     <string>Date: 
Time: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_BAC"/>
    <addaction name="actionRecent_BACs"/>
    <addaction name="actionOpen_BAC_Eff"/>
    <addaction name="actionRecent_BAC_Effs"/>
    <addaction name="actionOpen_BCM"/>
    <addaction name="actionRecent_BCMs"/>
    <addaction name="separator"/>
    <addaction name="actionSave_BAC"/>
    <addaction name="actionSave_BAC_as"/>
    <addaction name="actionSave_BCM"/>
    <addaction name="actionSave_BCM_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="actionSave_All_As"/>
    <addaction name="actionSave_BAC_Eff"/>
    <addaction name="actionSave_BAC_Eff_as"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_BAC">
   <property name="text">
    <string>Open BAC</string>
   </property>
  </action>
  <action name="actionRecent_BACs">
   <property name="text">
    <string>Recent BACs</string>
   </property>
  </action>
  <action name="actionOpen_BCM">
   <property name="text">
    <string>Open BCM</string>
   </property>
  </action>
  <action name="actionRecent_BCMs">
   <property name="text">
    <string>Recent BCMs</string>
   </property>
  </action>
  <action name="actionSave_BAC">
   <property name="text">
    <string>Save BAC</string>
   </property>
  </action>
  <action name="actionSave_BCM">
   <property name="text">
    <string>Save BCM</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionSave_BAC_as">
   <property name="text">
    <string>Save BAC As</string>
   </property>
  </action>
  <action name="actionSave_All_As">
   <property name="text">
    <string>Save All As</string>
   </property>
  </action>
  <action name="actionSave_BCM_As">
   <property name="text">
    <string>Save BCM As</string>
   </property>
  </action>
  <action name="actionOpen_BAC_Eff">
   <property name="text">
    <string>Open BAC_Eff</string>
   </property>
  </action>
  <action name="actionRecent_BAC_Effs">
   <property name="text">
    <string>Recent BAC_Effs</string>
   </property>
  </action>
  <action name="actionSave_BAC_Eff">
   <property name="text">
    <string>Save BAC_Eff</string>
   </property>
  </action>
  <action name="actionSave_BAC_Eff_as">
   <property name="text">
    <string>Save BAC_Eff as</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
